src/jdk/src/macosx/native/sun/awt/AWTView.m

```diff
@@ -55,10 +54,7 @@ -(BOOL) isCodePointInUnicodeBlockNeedingIMEvent: (unichar) codePoint;
 //#define EXTRA_DEBUG
 
 static BOOL shouldUsePressAndHold() {
-    static int shouldUsePressAndHold = -1;
```

Comment:
This looks like a behavior change.

Reply:
Keeping this behavior is probably not feasible because the APIs it depends on are not available on m1. The change only impacts macos 10.7, so proposing #357 as an alternative